src/test/resources/application-test.yml

```diff
@@ -20,11 +20,13 @@ hmpps.sqs:
 
 # Localstack settings
 aws:
-  region_name: eu-west-2
-  localstack_endpoint_url: "http://localhost:4566"
+  region-name: eu-west-2
   s3:
+    localstack-endpoint-url: "http://localhost:4566"
```

Comment:
`application-local.yml` has this `localstack-endpoint-url` key under `aws` rather than `aws.s3`

 `court-case-matcher` also has it under `aws` - might be better to standardise